packages/kbn-apm-synthtrace/src/cli/utils/synthtrace_worker.ts

```diff
@@ -25,16 +25,19 @@ export interface WorkerData {
   workerId: string;
   esUrl: string;
   version: string;
+  kibanaUrl: string;
 }
 
-const { bucketFrom, bucketTo, runOptions, esUrl, version } = workerData as WorkerData;
+const { bucketFrom, bucketTo, runOptions, esUrl, version, kibanaUrl } = workerData as WorkerData;
 
 async function start() {
   const logger = loggerProxy;
-  const assetsEsClient = getAssetsEsClient({
+
+  const entityEsClient = getEntityEsClient({
     concurrency: runOptions.concurrency,
     target: esUrl,
     logger,
+    kibanaTarget: kibanaUrl,
```

Comment:
I am not 100% convinced with the idea of passing the `kibanaTarget` to an `esClient`. We are polluting the ES client to do stuff which ideally a Kibana Client should do. I would recommend instead of passing this to the ES Client, you can generate a `EntityKibanaClient` just like its done for [APM](https://github.com/elastic/kibana/blob/736005d11b4dee7d76067793a9314526dd5105ac/packages/kbn-apm-synthtrace/src/cli/utils/bootstrap.ts#L25-L28)

And then also inject the `entityKibanaClient` to the bootstrap function so that in your scenario file you can call it the way you are doing it now

Reply:
Yeah, I was a bit lazy here 😆 .... will have it fixed.